learn/advanced/asynchronous_updates.md

```diff
@@ -95,8 +95,8 @@ Essentially, tasks are done in transactions. If the transaction fails or is kill
 You can use the `status` field returned by [the update route](/reference/api/updates.md) to determine if a process has been committed to MeiliSearch or not.
 
 - status: `enqueued` => Not yet begun. If MeiliSearch is killed and then restarted, the task will remain enqueued and be processed eventually.
-- status `processing` => In progress. If MeiliSearch is killed, there will be no consequences, since no part of the task has been committed to MeiliSearch. After restarting, Meilisearch will treat the task as `enqueued`.
-- status `done` => Completed. This action is done and is permanently added to your MeiliSearch instance. If you kill MeiliSearch, there will be no data loss; your database will remain exactly the same as before you killed MeiliSearch.
+- status `in_progress` => Processing. If MeiliSearch is killed, there will be no consequences, since no part of the task has been committed to MeiliSearch. After restarting, Meilisearch will treat the task as `enqueued`.
```

Comment:
I'm confused; isn't `in_progress` a status for Dumps, not updates?

According to your first comment, the three statuses described on this page should be `enqueued`, `processed`, and `failed`.

Reply:
```suggestion
- status `failed` => Not successful. If MeiliSearch is killed, there will be no consequences, since no part of the task has been committed to MeiliSearch.
```

Reply:
I thought you talked about dumps AND updates in this paragraph. Do you want to only mention the updates?

Reply:
The paragraph does not list the status but what happen if you kill meilisearch

Reply:
Good point, it's really unclear. I thought it was only speaking about updates, but clearly not since there is no `done` status for updates.

Reply:
BTW, I found where it mentions the possible values of `status`: https://docs.meilisearch.com/learn/advanced/asynchronous_updates.html#understanding-updates

Reply:
In this case, right above this list it says "You can use the status field returned by the update route to determine if a process has been committed to MeiliSearch or not." Since the update route doesn't work for Dumps, I think it should just be updates in this paragraph, after all.

Reply:
What should I do then? 🙂
Would you rather I open an issue?

Reply:
Yes, thanks! :)

Reply:
#877